frontend/components/common/formatters/TimeTill.vue

```diff
@@ -40,10 +50,15 @@ export default Vue.extend({
         calculateTime(): void {
             if (this.parsedDate) {
                 const now = new Date();
-                this.timeTill = formatInterval(now, this.parsedDate);
-
-                const duration = this.parsedDate.getTime() - now.getTime();
-                this.isEndingSoon = duration < ENDING_SOON_THRESHOLD && duration > 0;
+                if (this.isCountUp) {
+                    ++this.secondsElapsed;
+                    const newDate = new Date(now.getTime() + this.secondsElapsed * 1000);
```

Comment:
Why is `secondsElapsed` needed? 

The whole purpose of `parsedDate` is to not rely on `calculateTime` being called every second (which is not guaranteed by the browser). So why do you introduce a completely different logic to calculate time for `isCountUp` and completely ignore value of the `parsedDate` prop when in actuality you only need to change formatting?